.gitignore

```diff
@@ -1,49 +1,63 @@
-######################
-## Shared with linters
-######################
+#################################
+## Shared between git and linters
+#################################
 
-# Logs
+## Logs
```

Comment:
`##` is for comments, `#` is for commands that can be commented out to run. e.g.:

```
## Hope you know what you’re doing!
# rm -rf /my-project
```